operations/configuration_management/NCN_Node_Personalization.md

```diff
@@ -29,62 +66,151 @@ The following procedure describes how to correctly edit the `sat bootprep` files
     - name: ncn-personalization
     ```
 
-1. In the case of an upgrade, execute the following two substeps:
+### 2. Edit CPE and Analytics layers
+
+This section is required if this procedure is being performed as part of a CSM upgrade.
```

Comment:
I feel like this section wouldn't be required if this procedure was being performed as part of a CSM*-only* upgrade, I think when I wrote "in the case of an upgrade" I was thinking more like a full recipe upgrade.

Reply:
For sure. The CSM upgrade docs make that clear -- this page is only linked from the section that is for what we call the "standard upgrade", which is the non-CSM-only case. That said, it wouldn't hurt for me to be clearer here as well, to reduce the possibility of confusion. Good catch.